contracts/exchange/contracts/src/MixinWrapperFunctions.sol

```diff
@@ -74,7 +72,11 @@ contract MixinWrapperFunctions is
         returns (FillResults memory fillResults)
     {
         // ABI encode calldata for `fillOrder`
-        bytes memory fillOrderCalldata = _abiEncodeFillOrder(
+        bytes memory fillOrderCalldata = abi.encodeWithSelector(
+            // solhint-disable-next-line max-line-length
+            // bytes4(keccak256("fillOrder((address,address,address,address,uint256,uint256,uint256,uint256,uint256,uint256,bytes,bytes,bytes,bytes),uint256,bytes)"))
+            // = 0x9b44d556
+            0x9b44d556,
```

Comment:
Same as above - might want to use the constant from `LibExchangeSelectors` here.